src/main/java/org/symphonyoss/symphony/messageml/elements/DatePicker.java

```diff
@@ -44,10 +43,10 @@
   private static final String HIGHLIGHTED_DATE_PRESENTATION_ATTR = "data-highlighted-date";
   private static final String FORMAT_PRESENTATION_ATTR = "data-format";
 
-  private static final Range<Integer> ALLOWED_DAYS = Range.between(0, 6);
-
   private final ObjectMapper mapper;
 
+  private static final String dateFormatAllowed = "^[0-9Mdy\\/. -]+$";
```

Comment:
should we name it DATE_FORMAT_ALLOWED to comply with Java coding style?

Reply:
yes. Changed